### ISSUE TITLE
#67-make-red-dot-menu

### ISSUE THREAD

**Issue by Jkuras** [thavythach/PS-Tour-Guide#67]
make a lil interface that can place a phone call or send some kind of form email to red dot/campus security

**Comment by thavythach** [comment]
utilize the bar at the top where name of app is displayed. Use it to our advantage

**Comment by thavythach** [comment]
requires #72

**Comment by Jkuras** [comment]
yup, im going to write a toolbar xml that we can add to every activity with some buttons on it
